jep/1/README.adoc

```diff
@@ -276,17 +298,26 @@ let's take a high-level look at how JEP might go.
   When Andrea believes the JEP is stable,
   addresses all major design and scope questions,
   and represents the consensus of the community,
-  she then asks the <<Reviewer>>, in this case the <<BDFL Delegate>> Kelly,
-  to review the JEP for Acceptance.
-
-. **<<review, Review>>** - Kelly reviews the JEP
-  and any related discussions and implementation.
-  Kelly agrees with Andrea that consensus has been reached regarding the JEP
+  she then asks the <<Jenkins community>> to review the JEP for Acceptance.
+  This happens in the main Jenkins developer list,
+  and the message should reference the JEP discussion channel.
+
+. **<<review, Review>>** - any interested members of <<Jenkins community>> review the JEP
+  and any related discussions and implementation. 
+  Alternatively, if the <<Reviewer>> role had been delegated to another entity then they would also perform a review. 
+  All Reviewers agree with Andrea that consensus has been reached regarding the JEP
   and that the implementation is far enough along to enusure that
   the design is stable and complete.
-  Kelly marks the JEP as an "<<accepted, Accepted>>" JEP.
-
-. **<<accepted, Accepted Status>>** - Andrea and other contributors
+  This decision is made in a publicly traceable form.
+  Andrea announces the decision in the discussion channel defined for the JEP,
+  and explicitly sets the final response timeout (see <<accepted>>),
+  e.g. by saying "this JEP might be accepted in 7 calendar days unless there is unaddressed negative feedback".
```

Comment:
```suggestion
  e.g. by saying "this JEP will be accepted in 7 calendar days unless there is additional feedback".
```

Reply:
Do you agree with the comment @bitwiseman ?